dapps/dapps.json

```
@@ -268,6 +268,33 @@
       "categories": [
         "crowdloans"
       ]
+    },
+    {
+      "name": "DappRadar",
+      "icon": "https://raw.githubusercontent.com/nova-wallet/nova-utils/master/icons/dapps/color/DappRadar.svg",
+      "url": "https://dappradar.com/",
+      "categories": [
+        "utilities"
+      ]
+    },
+    {
+      "name": "TofuNFT",
+      "icon": "https://raw.githubusercontent.com/nova-wallet/nova-utils/master/icons/dapps/color/TofuNFT.svg",
+      "url": "https://tofunft.com/",
+      "categories": [
+        "nft",
+        "evm"
+      ]
+    },
+    {
+      "name": "RomeDAO",
+      "icon": "https://raw.githubusercontent.com/nova-wallet/nova-utils/master/icons/dapps/color/RomeDAO.svg",
+      "url": "https://romedao.finance/",
+      "categories": [
+        "defi",
+        "staking",
+        "evm"
+      ]
     }
```

```suggestion
    }
```
Looks like we shouldn't add this DApp as it sometimes does not work properly.

---

That comment was about wallet connection and that we need do it twice to success.
@leohar @antonkhvorov 
so guys, what would we do with that ? add or not?